### PR TITLE
typo-fix

### PR DESCRIPTION
**Pull request by aleksandrOranskiy**
Fixes: #nnn 
**Related**: _Relevant tracking issues, for context_
**Merge before/after**: _Dependent or prerequisite PRs_

**Description**
It's just a small typo fix.

**User facing changes (remove if N/A)**

**Follow-up Work (remove if N/A)**

### CODE REVIEW

File: docs/content/en/docs/pipeline-stages/builders/custom.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Skaffold will pass in the following additional environment variables for cluster
 
 **Configuration**
 
-To configure custom build script locally, in addition to adding a [`custom` field](#configuration) to each corresponding artifact in the `build`, add `cluster` to you `build` config.
+To configure custom build script in cluster, in addition to adding a [`custom` field](#configuration) to each corresponding artifact in the `build`, add `cluster` to you `build` config.
```
Comment (nkubala) [inline docs/content/en/docs/pipeline-stages/builders/custom.md:77]: :) 
```suggestion
To configure custom build script in cluster, in addition to adding a [`custom` field](#configuration) to each corresponding artifact in the `build`, add `cluster` to your `build` config.
```

**Review by nkubala — CHANGES_REQUESTED** [review]
@aleksandrOranskiy thanks for the fix! would you mind signing our CLA using the email you used to make this commit? i'll retrigger our integration tests to get CI green on here and then we can get this merged

**Comment by aleksandrOranskiy** [comment]
@googlebot I signed it!

**Review by MarlonGamez — APPROVED** [review]
lgtm! Thanks @aleksandrOranskiy :)

**Pull request merged**